lib/util/mongo.js

```diff
@@ -128,7 +129,34 @@ function getAdminDbUrl(mongodbUrl, adminAuth) {
   return mongoUriParser.format(parsedMongoConn);
 }
 
+/**
+ * Check whether an app has a dedicated database for an environment.
+ * @param {object} params.
+ * @param {string} params.domain - The domain name.
+ * @param {string} params.appname - The app id.
+ * @param {string} params.environment - The environment name.
+ */
+function hasDedicatedDb(params, cb) {
+  logger.trace(params, 'is dedicated database');
+
+  MongoClient.connect(fhmbaasMiddleware.config().mongoUrl, function(err, db) {
```

Comment:
Do we need to create a new connection here? Can't we reuse the mongoose connection from here https://github.com/feedhenry/fh-mbaas/blob/master/fh-mbaas.js#L257 ?